docs/data/query.md

```diff
@@ -202,6 +202,31 @@ See the [MongoDB Atlas Documentation](https://www.mongodb.com/docs/atlas/data-fe
 
 For information on connecting to your Atlas instance from other MQL clients, see the MongoDB Atlas [Connect to your Cluster Tutorial](https://www.mongodb.com/docs/atlas/tutorial/connect-to-your-cluster/).
 
+#### Query by date
+
+When using MQL to query your data by date or time range, you can optimize query performance by avoiding the MongoDB `$toDate` expression, using the [BSON `date` type](https://www.mongodb.com/docs/manual/reference/bson-types/#date) instead.
+
+For example, you could use the following to query a dataset over a date range, specifying a target start timestamp in BSON date format, and using the `Date()` constructor to use the current time as the end timestamp:
```

Comment:
```suggestion
For example, you could use the following to query a dataset over a date range, specifying a target start timestamp in BSON date format, and using the JavaScript `Date()` constructor to use the current time as the end timestamp:
```
Thoughts? Just to make it clear that this is specific to JavaScript/the shell, since to construct a date type in another language they will need to use the language-specific constructs

Reply:
Right on, this is indeed continuing the `mongoh` example, so JS it is. Clarified this a bit, thank you!